lib/storage/keyval/system.go

```diff
@@ -50,3 +50,33 @@ func (b *backend) GetSELinux() (enabled bool, err error) {
 func (b *backend) SetSELinux(enabled bool) error {
 	return b.upsertVal(b.key(systemP, seLinuxP), &enabled, forever)
 }
+
+// GetNodeAddr returns the current node advertise IP
+func (b *backend) GetNodeAddr() (addr string, err error) {
```

Comment:
I realize this is probably just part of the port and already this way on other branches, I imagine it applies to just the local backend. I just find it funny, because the other backend is etcd, in which case each node would clobber others writes.

Not asking for any changes, just something I found interesting.

Reply:
Agree - moved the system metadata APIs to a separate interface in https://github.com/gravitational/gravity/commit/c059b97e